pyomo/contrib/pyros/uncertainty_sets.py

```diff
@@ -3418,7 +3433,7 @@ def type(self):
     @property
     def scenarios(self):
         """
-        list of tuples : Uncertain parameter realizations comprising the
+        list of tuple : Uncertain parameter realizations comprising the
```

Comment:
Can we make this a formal return type (i.e., `list[tuple[...]]`)?

(Same with other `scenarios()` below)